test/multiapi/Expected/AcceptanceTests/MultiapiWithSubmodule/multiapiwithsubmodule/submodule/_operations_mixin.py

```diff
@@ -94,21 +133,69 @@ def begin_test_lro_and_paging(
         mixin_instance._deserialize = Deserializer(self._models_dict(api_version))
         return mixin_instance.begin_test_lro_and_paging(client_request_id, test_lro_and_paging_options, **kwargs)
 
+    @_InspectArgsForApiVersion()
+    def test_different_calls(
+        self,
+        greeting_in_english,  # type: str
+        greeting_in_chinese=None,  # type: Optional[str]
+        greeting_in_french=None,  # type: Optional[str]
+        **kwargs  # type: Any
+    ):
+        """Has added parameters across the API versions.
+
+        :param greeting_in_english: pass in 'hello' to pass test.
+        :type greeting_in_english: str
+        :param greeting_in_chinese: pass in 'nihao' to pass test.
+        :type greeting_in_chinese: str
+        :param greeting_in_french: pass in 'bonjour' to pass test.
+        :type greeting_in_french: str
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: None, or the result of cls(response)
+        :rtype: None
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        **New in API version 2.0.0**
```

Comment:
@johanste the reason this is not `:: versionadded` annotation was bc I chatted with @lmazuel , and the versionadded annotation is about package version, not API version, so the generated message could be unclear.

Here is sphinx rendering with ::versionadded
![image](https://user-images.githubusercontent.com/43154838/94969133-2aa38400-04d0-11eb-8294-596662145367.png)

Reply:
I was thinking that we can add which API version a parameter/value was added to as a comment to the directive:

```
.. versionadded:: 3
   Supported in service API version 6.78 and later.
```

Reply:
Sorry @johanste I'm not understanding the example you gave. Wouldn't that render as
"_New in version 3_: Supported in service API version 6.78 and later."?

Reply:
Yeah, you are probably correct. It is unfortunate if we cannot also tell which package version APIs were introduced in (my end-goal here was to *not* have to have the annoying "version" drop-down in our documentation - or at least minimize the need to use it). But it probably isn't at the top of the list of doc improvements we need to make....

Reply:
this comment is both relieving, and foreboding 😆